README.md

```diff
@@ -114,45 +114,15 @@ Version: 3
-
-- Asking people to pay for plugins when the `termux-app` at installation time is broken due to repository issues and has bugs is unethical.
-
-- Old versions don't have proper logging/debugging and crash report support. Reporting bugs without logs or detailed info is not helpful in solving them.
-
-- It's also easier for us to solve package related issues and provide custom functionality with app updates, which can't be done if users continue using old versions. For example, the [bintray shutdown](https://github.com/termux/termux-packages/wiki/Package-Management) causing package install/update failures for new Play Store users is/was not an issue for F-Droid users since it is being shipped with updated bootstrap and repo info, hence no reported issues from new F-Droid users.
-</details>
-
-##
+Currently, Google Play will try to update installations away from F-Droid ones. Updating will still fail as [sharedUserId](https://developer.android.com/guide/topics/manifest/manifest-element#uid) has been removed. This will be fixed in a `0.118.1` version update on F-Droid shortly, which will update the version code so that Google Play does not considers its version of the app newer. Meanwhile, to prevent PlayStore from attempting to download and then fail to install the Google Play releases over existing installations, you can open the Termux apps pages on Google Play and then click on the 3 dots options button in the top right and then disable the Enable auto update toggle. However, the Termux apps updates will still show in the PlayStore app updates list.
 
+There is currently a build of Termux available on Google Play for Android 11+ devices, with extensive adjustments in order to pass policy requirements there. This is under development and has missing functionality and bugs (see [#4000](https://github.com/termux/termux-app/discussions/4000) for more information and [here](https://github.com/termux-play-store/) for status updates) compared to the stable F-Droid build, which is why most users who can should still stay on the F-Droid build.
 
+If you want to help out with testing the Google Play build (or cannot install Termux from other sources), be aware that it's built from a separate repository (https://github.com/termux-play-store/) - be sure to report issues [there](https://github.com/termux-play-store/termux-issues/issues/new/choose), as any issues might very well be specific to that repository and not relevant for the main Termux ones.
```

Comment:
I think these two last paragraphs can be shortened, playstore specific info should go in the playstore repo. The play-store app repo is built from a quite different code base, and is for most purposes a fork of github.com/termux/termux-app at this point. As an alternative I propose:

A Termux app built from different sources is available from Google Play for Android 11+ devices. It is under development and has missing functionality and bugs compared to F-Droid and GitHub releases (see [#4000](https://github.com/termux/termux-app/discussions/4000)). Issues with PlayStore app are off-topic here and should be reported in a [separate organisation](https://github.com/termux-play-store/termux-issues/issues/new/choose).